frontend/src/components/InGame/helper/GamePlaying.vue

```diff
@@ -1,6 +1,17 @@
 <template>
-  <div v-if="isJudge || submitted">
+  <div v-if="isJudge || hasSubmitted">
     <h6 class="mt-5 text-center">Waiting for other players to submit ...</h6>
+    <div class="d-flex flex-nowrap justify-content-center pb-4 submission-wrap">
+      <div
+        v-for="i in playerCount - 1"
+        :key="i"
+        class="submission-card play-card white-card"
+        :class="{ visible: !!submissions[i - 1] }"
+        :style="`--randVal: ${getRandomDegrees(i)}; z-index: ${i};`"
```

Comment:
Maybe use object syntax?

Reply:
Might be cleaner, but doesn't really matter in that case I'd say.